### DISCUSSION TITLE
How to stop Directus sending 304 responses

### DISCUSSION THREAD

**Discussion by thepurpleblob** [directus/directus#13080] [Archive]
Log shows...

```
18:01:27 ✨ request completed GET 304 /items/Calendar?limit=-1 6ms
18:01:46 ✨ request completed GET 304 /items/Calendar?limit=-1 5ms
18:08:09 ✨ request completed GET 304 /items/Calendar?limit=-1 25ms
```

They appear to be breaking my (axios) client on some browsers. Is there some way to stop Directus responding with a 304?

### To Reproduce

I can't - but the issue has been reported by multiple users

<details>
<summary>More info</summary>

### Errors Shown

client fails because it doesn't appear to be able to handle the 304

### What version of Directus are you using?

9.6.0

### What version of Node.js are you using?

latest

### What database are you using?

SQLite

### What browser are you using?

n/a

### How are you deploying Directus?

Docker

</details>

**Comment by azrikahar** [comment]
Granted I'm not exactly sure what is breaking here, since 304 should mean it's cached 🤔 or perhaps you mean breaking in terms of stale data?. It'd be very helpful if there's a more detailed error log from your user, if possible.

You may be able try making sure cache is turned off: https://docs.directus.io/configuration/config-options/#cache

**Comment by thepurpleblob** [comment]
I should probably explain more. I'm getting inconclusive errors from axios on the client. In fact, it's not showing any status codes or errors at all. I'd quite like to eliminate the 304 simply by disabling that behaviour in Directus if that's possible to do. 

Setting CACHE_ENABLED to false does not stop the 304s but I may be misunderstanding something.

**Comment by rijkvanzanten** [comment]
Directus itself doesn't send 304s, so I think this is either the browser telling you the resource was cached, or some other proxy/cdn server in between

**Comment by rhemlock7** [comment]
Was there a fix found for this?

**Comment by thepurpleblob** [comment]
I never worked out what was going wrong. I think I coded around it but it was quite a while ago. 

If it makes any difference, I have Directus running in a Docker container and then reverse-proxied by Apache.

**Reply by rhemlock7** [reply]
Ah ok got it. I'm running into the same issue now with getting 304's and struggling to find a fix. I'm noticing it in my Directus system logs and other logs
